### DISCUSSION TITLE
QR codes in Expression 3a

### DISCUSSION THREAD

**Discussion by HansBuob1968** [manuelbl/SwissQRBill#87] [Q&A]
Ist it prossoble to generate qr-code in expression 3a, is it planed?

See attached file
![3a](https://github.com/manuelbl/SwissQRBill/assets/146735324/49896ded-ffc7-4df8-a507-c7437705e2ea)

Thanks

**Comment by manuelbl** [comment]
Yes, that's possible. Did you run into any specific problems?

You can try the library at https://codecrete.net/qrbill.

**Comment by HansBuob1968** [comment]
Thank you, with the help of this library url i was able to find the problem. This was on my side. I was testing it with a qr-iban, but a 3 a expression must be a non qr iban.
Thank you, also for the fast response

Greeting Hans

**Comment by manuelbl** [comment]
You are welcome.

The validation rules related to QR-IBAN vs non-QR-IBAN are indeed a bit tricky.
